domains/authorization/Authorization.API/API/Data/IUnitOfWork.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using API.Models;
+
+namespace API.Data;
+
+public interface IUnitOfWork : IDisposable
+{
+    IRepository<User> Users { get; }
+    IRepository<Organization> Organizations { get; }
+    IRepository<Affiliation> Affiliations { get; }
+    IRepository<Client> Clients { get; }
+    IRepository<Consent> Consents { get; }
+    int Complete();
```

Comment:
It's called SaveAsync in other places we do this.

Reply:
No longer applicable